Coding/fw/lib/Test/WString.h

```diff
@@ -489,6 +489,18 @@ String(unsigned long value, unsigned char base) {
     *this = buf;
 }
 
+bool isEmpty()
+{
+    bool isEmpty = false;
```

Comment:
that is unecessary long as all it does is wrapping length() !=0

Why not 

bool isEmpty() const { return 0 != length(); }

also mind the const keyword here.
doxygen also missing